Code/Framework/AzToolsFramework/AzToolsFramework/ActionManager/Menu/EditorMenuBar.h

```diff
@@ -35,10 +35,11 @@ namespace AzToolsFramework
         // Returns the pointer to the menu bar.
         QMenuBar* GetMenuBar();
         const QMenuBar* GetMenuBar() const;
-
-    private:
+        
```

Comment:
Looks like an extra tab space here?

Reply:
Fixing in the follow-up PR if this one passes AR :)